dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -0,0 +1,166 @@
+[[nexus-maven-tutorial]]
+= Setting Up a Nexus Mirror for Maven
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[nexus-introduction]]
+== Introduction
+
+While developing your application with Java and Maven, you will most likely be
+building many times. In order to shorten the build times of your pods, Maven
+dependencies can be cached in a local Nexus repository. This tutorial will
+guide you through creating a Nexus repository on your cluster.
+
+This tutorial assumes that you are working with a project that is already set
+up for use with Maven. If you are interested in using Maven with your Java
+project, it is highly recommended that you look at
+link:https://maven.apache.org/guides/getting-started/index.html[their guide].
+
+In addition, be sure to check your application's image for Maven mirror
+capabilities. Many images that use Maven have a `*MAVEN_MIRROR_URL*` environment
+variable that you can use to simplify this process. If it does not have this
+capability, read
+link:https://books.sonatype.com/nexus-book/reference/config.html[the Nexus documentation]
+to configure your build properly.
+
+[[nexus-setting-up-nexus]]
+== Setting up Nexus
+
+Use `new-app` to download and deploy the official Nexus Docker image.
+
+----
+oc new-app sonatype/nexus
+----
+
+After the build has finished, run:
+
+----
+oc get svc
+----
+
+The output should resemble:
+
+----
+NAME      CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+nexus     172.30.170.249   <none>        8081/TCP   6s
+----
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081. The server may take a bit of time to start, be patient!
```

Comment:
sending users to the cluster ip is a dubious practice since users of most clusters (like online) won't be able to reach the cluster ip.  I think it would be better to add an "oc expose" command which will create a route and then have them use the route to access the server.